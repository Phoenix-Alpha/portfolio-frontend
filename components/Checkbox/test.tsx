import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Checkbox } from '.'

describe('<Checkbox />', () => {
  it('should renders correct <Checkbox />', () => {
    const wrapper = shallow(<Checkbox />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
