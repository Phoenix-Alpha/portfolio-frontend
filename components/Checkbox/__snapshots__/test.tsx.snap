// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> should renders correct <Checkbox /> 1`] = `
<label
  className="flex flex-row items-center select-none"
>
  <input
    className="appearance-none"
  />
  <CheckIcon
    className="mr-2 disabled:bg-gray-5"
  />
</label>
`;
